website/docs/docs/cloud/dbt-cloud-ide/develop-in-the-cloud.md

```diff
@@ -6,8 +6,7 @@ sidebar_label: Develop in the IDE
 tags: [IDE]
 ---
 
-The dbt Cloud integrated development environment (IDE) is a single interface for building, testing, running, and version-controlling dbt projects from your browser. With the Cloud IDE, you can compile dbt code into SQL and run it against your database directly. The IDE leverages the open-source [dbt-rpc](/reference/commands/rpc) plugin to recompile only the changes made in your project.
-
+The dbt Cloud integrated development environment (IDE) is a single interface for building, testing, running, and version-controlling dbt projects from your browser. With the Cloud IDE, you can compile dbt code into SQL and run it against your database directly. The IDE leverages the dbt-server, making the IDE faster and more reliable.
```

Comment:
The second sentence here may be worth removing or amending.

We removed dbt-server from IDE v1.2 and instead are using a closed-source fork called `ide-server`.
We also are not yet able to claim that it is faster or more reliable than local development 😓 Although that is certainly a goal!